### ISSUE TITLE
Crab and ice demon timers, shaman acid splat, less specific whitelist for raids

### ISSUE THREAD

**Issue by MagicfTail** [runelite/runelite#979]
- Crab and ice demon burner timers

- Shaman acid splat marker

- Less specific layout whitelist, fx. mark crab-rope raids

**Comment by Infinitay** [comment]
I don't your second suggestion will be implemented because we had a plugin before for some AoE/Projectile attacks and Jagex asked us to remove it.

**Comment by rune3132** [comment]
I had a feeling something similar was removed, but I wasn't completely sure. Decided to add it anyways as shamans is probably the biggest raids-noob killer, and this would make it a lot more manageable.

**Comment by Kamielvf** [comment]
I don't think ice demon burner timers can be done properly since it depends on how much kindling was put in and I wouldn't be able to get accurate timings when other players put kindling in. Unless I am misunderstanding the suggestion, in that case please let me know :)

As for the crab timer I currently already have [this](https://puu.sh/zHlBK/b092b08e2b.gif).

I will not add a marker for shaman acid attacks because of what @Infinitay mentioned.

**Comment by rune3132** [comment]
Yeah, the burn timer would probably only be doable for solo raids, and it isn't really a problem there since you dump a predetermined amount anyways.

The stun timer looks great. The stun time was seemingly random when I tried to measure it, so I imagine it isn't going to be completely precise, not that it really matters as long as it isn't too long.
I also thought about adding an icon of some sort to the coloured crystals, showing the required attack style, but I don't know if that is too much hand holding as you should be able to get it right pretty much after doing that room once.

The shaman attack is completely fine as it would probably be a bit too powerful anyway.
Keep up the great work :)

**Comment by Kamielvf** [comment]
Yeah the timer can be slightly off at times but I might be able to patch a perfect timer in later if I manage to find out their behaviour.
There's some more things I'm working on for the raids plugin so I can't say when exactly it will be in the client.
